src/systemd/sd-dhcp-static-lease.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+#ifndef foosddhcpstaticleasehfoo
+#define foosddhcpstaticleasehfoo
+
+#include "_sd-common.h"
+#include <netinet/in.h>
+
+_SD_BEGIN_DECLARATIONS;
+
+typedef struct sd_dhcp_static_lease sd_dhcp_static_lease;
+
+int sd_dhcp_static_lease_new(sd_dhcp_static_lease **ret);
+int sd_dhcp_static_lease_set_client_id_by_mac(sd_dhcp_static_lease *lease, const uint8_t *mac_addr, size_t mac_addr_size);
+int sd_dhcp_static_lease_unset_client_id(sd_dhcp_static_lease *lease);
+int sd_dhcp_static_lease_is_address_set(sd_dhcp_static_lease *lease);
+int sd_dhcp_static_lease_set_address(sd_dhcp_static_lease *lease, const struct in_addr *address);
+int sd_dhcp_static_lease_unset_address(sd_dhcp_static_lease *lease);
```

Comment:
I do not think exposing `struct sd_dhcp_static_lease` and `sd_dhcp_static_lease_xxx()` is not good idea now.

How about simply `sd_dhcp_server_add_static_lease()` takes IP and MAC addresses?

Reply:
I removed the sd-dhcp-static-lease files completely. I think that resolves this comment.